manifests/init.pp

```diff
@@ -171,4 +171,11 @@
   Exec['cpinit'] -> Exec['foreman-rake-db:seed']
 
   User<|title == apache|>{groups +> $user_groups}
+
+  # lint:ignore:spaceship_operator_without_tag
+  Certs::Ssltools::Certutil <| |> ~> Service['qpidd']
```

Comment:
Can't we catch this in some other way? I'd think the ```Class['certs::qpid']``` would also take care of this.